controllers/toolchainstatus/toolchainstatus_controller_test.go

```diff
@@ -1724,6 +1724,15 @@ func TestGenerateUnreadyNotificationContent(t *testing.T) {
 	})
 }
 
+func TestRemoveSchemeFromUrl(t *testing.T) {
+	t.Run("test proxy url domain extraction", func(t *testing.T) {
+		domain, err := removeSchemeFromUrl("https://api-toolchain-host-operator.apps.stone-stg-host.qc0p.p1.openshiftapps.com")
+		require.Equal(t, "api-toolchain-host-operator.apps.stone-stg-host.qc0p.p1.openshiftapps.com", domain)
+		require.Nil(t, err)
```

Comment:
Minor thing but our tests are usually sectioned by 'given', 'when', 'then' comments. With them it's easier to distinguish between test setup, what's being tested and the assertions.
```suggestion
		// when
		domain, err := removeSchemeFromUrl("https://api-toolchain-host-operator.apps.stone-stg-host.qc0p.p1.openshiftapps.com")
		// then
		require.Equal(t, "api-toolchain-host-operator.apps.stone-stg-host.qc0p.p1.openshiftapps.com", domain)
		require.Nil(t, err)
```